src/main/java/org/openstreetmap/atlas/geography/atlas/complete/CompleteRelation.java

```diff
@@ -36,6 +39,9 @@
     private Long osmRelationIdentifier;
     private Set<Long> relationIdentifiers;
 
+    @Delegate
+    private final TagChangeDelegate tagChangeDelegate = TagChangeDelegate.newTagChangeDelegate();
```

Comment:
Could that have any impact on Serialization? Otherwise I would make it transient here!

Reply:
The classes (and interfaces) in question have been marked Serialziable.